fem/fe_coll.cpp

```diff
@@ -394,6 +394,50 @@ FiniteElementCollection *FiniteElementCollection::New(const char *name)
    return fec;
 }
 
+FiniteElementCollection *FiniteElementCollection::NewH1(const int order,
+                                                        const int dim,
+                                                        const bool NURBS)
+{
+   if (NURBS)
+   {
+      return new NURBSFECollection(order);
+   }
+   return new H1_FECollection(order, dim);
+}
+
+FiniteElementCollection *FiniteElementCollection::NewL2(const int order,
+                                                        const int dim,
+                                                        const bool NURBS)
+{
+   if (NURBS)
+   {
+      return new NURBSFECollection(order);
+   }
+   return new L2_FECollection(order, dim);
+}
+
+FiniteElementCollection *FiniteElementCollection::NewHDiv(const int order,
+                                                          const int dim,
+                                                          const bool NURBS)
+{
+   if (NURBS)
+   {
+      //return new NURBSFECollection(order);
```

Comment:
Not supported yet?

Reply:
Hdiv and Hcurl are available now because of merge of #3999